lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -910,7 +910,8 @@ void ResourceMap::loadDefaultConfiguration()
   addAsUnsignedInteger("FejerAlgorithm-DefaultMarginalIntegrationPointsNumber", 64);
 
   // IteratedQuadrature parameters //
-  addAsScalar("IteratedQuadrature-MaximumError",    1.0e-7);
+  addAsScalar("IteratedQuadrature-MaximumError", 1.0e-7);
+  addAsString("IteratedQuadrature-Rule", "G3K7");
```

Comment:
Please document these new ResourceMap keys in the doc, otherwise only extremely informed users will be able to use them.